catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/sharing/ShareableMetacardAttributes.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.sharing;
```

Comment:
❓ Why are these being defined in the UI module instead of with all the other taxonomy attributes? I'd like to talk through some of the use-cases and our naming conventions a little more.

Reply:
I think that we've had non-taxonomy attributes (such as attributes exclusive to workspace or query metacards) in this bundle since Intrigue was added to the baseline.  I don't think we consider them part of the taxonomy.